app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -149,8 +150,9 @@ public void onGet(Object item) {
                     ActivityFeed feed = (ActivityFeed) item;
 
                     if (isAdded()) {
-                        Crouton.makeText(getActivity(), R.string.cached_content,
```

Comment:
Above in `MainActivity` you created `mContentFrameLayout` and put the Snackbar into it. Why it is different here? Thank you.